node/sc/sub_bridge_handler.go

```diff
@@ -342,6 +376,39 @@ func (sbh *SubBridgeHandler) LocalChainHeadEvent(block *types.Block) {
 	}
 }
 
+func (sbh *SubBridgeHandler) handleParentChainInvalidTxResponseMsg(msg p2p.Msg) error {
+	headerNum := sbh.subbridge.blockchain.CurrentHeader().Number
+	var invalidTxs []InvalidParentChainTx
+	if err := msg.Decode(&invalidTxs); err != nil && err != rlp.EOL {
+		return errResp(ErrDecode, "msg %v: %v", msg, err)
+	}
+	txPool := sbh.subbridge.GetBridgeTxPool()
+	for _, invalidTx := range invalidTxs {
+		if tx := txPool.Get(invalidTx.TxHash); tx != nil {
+			logger.Error("A bridge tx was not executed", "err", invalidTx.ErrStr,
+				"prevGasPrice", sbh.subbridge.bridgeAccounts.GetParentGasPrice(),
```

Comment:
Is this previous gas price?
If so, I think after one loop is executed once, this log will not be valid, because the parent gas price of `sbh.subbridge.bridgeAccounts` will be updated in L395.

Reply:
What a great point! The log `"prevGasPrice", sbh.subbridge.bridgeAccounts.GetParentGasPrice()` is actually the same with `"txGasPrice", tx.GasPrice().Uint64())`. So, I decided remove the former one. Hopefully, I'd like to request you to review my PRs (in the future) when you are available.